plugins/tar/receive-app

```diff
@@ -11,7 +12,10 @@ tar_receive_app() {
   if [[ ! -f "$DOKKU_ROOT/$APP/src.tar" ]]; then
     true
   else
-    dokku tar:build "$APP"
+    acquire_app_deploy_lock "$APP"
+    # shellcheck disable=SC2086
+    tar_build "$@"
+    release_app_deploy_lock "$APP"
```

Comment:
No way for us to move the lock to the function itself? Or maybe a function like `lock_app_build tar "$APP"`?